odoo/tools/translate.py

```diff
@@ -1490,11 +1579,14 @@ def load(self, fileobj, fileformat, lang, xmlids=None):
     def _load(self, reader, lang, xmlids=None):
         if xmlids and not isinstance(xmlids, set):
             xmlids = set(xmlids)
+        base_lang = get_base_lang(lang)
         for row in reader:
             if not row.get('value') or not row.get('src'):  # ignore empty translations
                 continue
             if row.get('type') == 'code':  # ignore code translations
                 continue
+            if row.get('lang', lang) not in (lang, base_lang):
+                continue
```

Comment:
Hi, I think the special case for es_419 is still not handled here. Could you have a check?

Reply:
It is, but GitHub is bugged.
If you look at Runbot, the last hash is not the same as on GitHub...

See https://discord.com/channels/678381219515465750/704997651833159831/1331269102731268211

I think I'll have to open a new PR (tomorrow because I'm on leave today and don't have my computer handy)